test/full/t_cache.clj

```diff
@@ -3,12 +3,35 @@
             [full.core.sugar :refer :all]
             [full.async :refer :all]
             [full.cache :as cache]
-            [clojure.core.async :as async :refer [<!!]]))
+            [clojure.core.async :as async :refer [<!!]]
+            [taoensso.nippy :as nippy]))
+
+(def ^:private some-bytes
```

Comment:
This should really be named better. Is there some specific meaning to these particular bytes? Or are they really just random bunch of bytes?

Reply:
They are random (well, they spell `FullContact!`) but it was for a test of a random input of bytes nippy wouldn't parse lol.